start.sh

```diff
@@ -22,11 +22,15 @@ echo 'starting...'
 ./boot2docker.exe start
 echo
 
+echo 'IP address of docker VM:'
 ./boot2docker.exe ip
+echo
 
-echo 'connecting...'
-./boot2docker.exe ssh
+echo 'setting environment variables ...'
+./boot2docker.exe shellinit | sed  's,\\,\\\\,g' # eval swallows single backslashes in windows style path
+eval $(./boot2docker.exe shellinit 2>/dev/null | sed  's,\\,\\\\,g')
```

Comment:
I feel like maybe this should be a fix in boot2docker-cli too (handling backslashes properly).